netci.groovy

```diff
@@ -122,7 +129,7 @@ def static isLongGc(def scenario) {
 
 def static setTestJobTimeOut(newJob, scenario) {
     if (isGCStressRelatedTesting(scenario)) {
-        Utilities.setJobTimeout(newJob, 1440)
+        Utilities.setJobTimeout(newJob, 2880)
```

Comment:
I assume we started blowing past the previous timeout due to newly-added tests.  Any reason to believe 2 days is going to be sufficient?  I'm inclined to pick a bigger number like 3 or 4 days now so that we have more confidence we won't hit timeouts in the next round.  If that turns out to be overkill, we can pull it in later, but at least we didn't lose a round.